twistr/score.go

```diff
@@ -0,0 +1,126 @@
+package twistr
+
+type ScoreLevel int
+
+const (
+	Nothing ScoreLevel = iota
+	Presence
+	Domination
+	Control
+)
+
+const WIN int = -1
+
+func VPAward(level ScoreLevel, r Region) int {
```

Comment:
Just punting on SEAsia?

It should probably still appear in this switch with a call to its own scoring function. Stupid SEAsia.